docs/doxygen/config.doxyfile

```diff
@@ -958,7 +957,7 @@ IMAGE_PATH             = ./docs/doxygen/images/mqtt_basic_tls.png \
                          ./docs/doxygen/images/mqtt_serializer.png \
                          ./docs/doxygen/images/mqtt_subscription_manager.png \
                          ./docs/doxygen/images/retry_utils_flow.png \
-                         ./docs/doxygen/logging/logging_architecture.png \
+                         ./docs/doxygen/images/logging_architecture.png \
```

Comment:
Why can't we just delete all these lines and use `./docs/doxygen/images/`?

Reply:
Hygiene for next PR if possible?

Reply:
It doesn't work

Reply:
I wrote it doesn't work. But I might have mixing up case sensitivity.